provider/google/google.go

```diff
@@ -32,6 +32,7 @@ func (p *GoogleProvider) ConfigureRun(state *run.State) error {
 	}
 
 	state.RootURL = userData.RootURL
+	state.CloudProviderName = "gcp"
```

Comment:
For better or worse, we've been consistent about calling this "google" rather than "gcp", so let's keep doing that.